tests/app/dao/test_organisation_dao.py

```diff
@@ -383,3 +385,23 @@ def test_get_organisation_by_email_address_ignores_gsi_gov_uk(notify_db_session)
 
     found_org = dao_get_organisation_by_email_address('test_gsi_address@example.gsi.gov.uk')
     assert org == found_org
+
+
+def test_add_to_and_get_email_branding_pool_for_organisation(sample_organisation):
```

Comment:
I think it would be useful to  expand this to check that if there a third email branding that exists and is in the email branding pool of organisation 2 then it does not get included in the results for organisation 1 please

Reply:
We will add that in, will make the test more robust for sure :)

Reply:
Done now (thanks @BlessedDev ! )